lib/storage-handler.js

```diff
@@ -258,6 +258,9 @@ exports.download = function(provider, req, res, container, file, cb) {
       reader.on('error', function(err) {
         handleError(res, err);
       });
+      reader.on('end', function() {
```

Comment:
If I am not mistaken, the afterRemote hook is not invoked when there is an error with the remote-method right?